### DISCUSSION TITLE
Is it considered cheating to google a formula

### DISCUSSION THREAD

**Discussion by Nik-Sharp** [codewars/codewars.com#2556] [Q&A] [answered]
I use google when trying to debug an error, but I'm not sure if I can use it to find formulas. For example, I'm doing [Sums of Perfect Squares](https://www.codewars.com/kata/5a3af5b1ee1aaeabfe000084/train/csharp) right now. I want to find out how to find the closest perfect square to a number, so I want to know if I can google it.

**Comment by Blind4Basics** [comment, marked as answer]
yes, you can (nothing forbids you to do so anyway). Just keep in mind that google won't improve your skills, in the end, so googling everything isn't an actual solution, that's all.

cheers
